pygitops/_util.py

```diff
@@ -103,3 +104,17 @@ def push_error_present(push_info: PushInfo) -> bool:
     Check for presence of the error flag in the returned flags bitmask.
     """
     return bool(push_info.flags & push_info.ERROR)
+
+
+def is_git_repo(path: Path) -> bool:
+    """
+    Determine if a given path is a valid git repository.
+
+    :param path: Directory to determine is a directory
+    :return: True if the contents of a directory at given path contains a valid git repository
+    """
+    try:
+        _ = Repo(path).git_dir
+        return True
```

Comment:
```suggestion
        return bool(Repo(path).git_dir)
```

Reply:
Since `Repo(path).git_dir` resolves to to a string or throws an exception, it might be a little weird to cast it to a bool. But the assignment is not necessary (I don't think python optimizes out statements with "no effect" like this?)